### ISSUE TITLE
must set "ignore invalid ssl certs" option for tower

### ISSUE THREAD

**Issue by ikke-t** [mglantz/ansible-roadshow#58]
otherwise tower can't connect to self-signed ssl GitLab. This is the option:

/etc/tower/settings.py:
```
AWX_TASK_ENV['GIT_SSL_NO_VERIFY'] = 'True'
```

I tested this works.

**Comment by ikke-t** [comment]
This needs to be put into one of the tower yamls:

```
- lineinfile:
    path: /etc/tower/settings.py
    state: present
    regexp: '.*GIT_SSL_NO_VERIFY.*'
    line: AWX_TASK_ENV['GIT_SSL_NO_VERIFY'] = 'True'
  notify: restart-tower
```

I doubt that there is "restart-tower" notification, so perhaps just do it before starting tower for the first time.

**Comment by ikke-t** [comment]
If it's put in before the backup restore script, we need to make sure backup restore won't override it.

**Comment by ikke-t** [comment]
@mglantz where would this fit the best?

**Comment by pgustafs** [comment]
I think the backup script is just restoring the DB, but I would add it after and then a restart of the tower service

**Comment by mglantz** [comment]
Fixed in  986da3b
